src/commands/exportConnectionProfileConnectedCommand.ts

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
don't need this class as can do the samething from where you register the command